lib/WP_Auth0_LoginManager.php

```diff
@@ -150,9 +150,10 @@ public function init_auth0() {
 
     // Catch any incoming errors and stop the login process
     // See https://auth0.com/docs/libraries/error-messages
-    if ( $this->query_vars( 'error_description' ) ) {
+    if ( $this->query_vars( 'error' ) || $this->query_vars( 'error_description' ) ) {
```

Comment:
I think the `error_description` might be missing sometimes (AFAIK is optional). In any case, because `error` is always present that condition should suffice.